01-main/manifest

```diff
@@ -56,7 +56,6 @@ duf
 dustracing2d
 element-desktop
 enpass
-exodus
```

Comment:
Preferable to mark as deprecated ('s/exodus/#exodus/') so users get to know it is no longer being kept updated by `deb-get` and they should consider uninstalling.